warehouse/models/mart/gtfs_schedule_latest/dim_gtfs_datasets_latest.sql

```diff
@@ -0,0 +1,30 @@
+WITH
+unfiltered_entries_latest AS (
+    {{ get_latest_schedule_data(
+    table_name = ref('dim_gtfs_datasets'),
+    clean_table_name = 'dim_gtfs_datasets'
+    ) }}
+),
+
+bridge_schedule_dataset_for_validation AS (
+    SELECT * FROM {{ ref('bridge_schedule_dataset_for_validation') }}
+    where _is_current
+),
+
+dim_gtfs_datasets_latest AS (
+    SELECT
+        unfiltered_entries_latest.name,
+        unfiltered_entries_latest.type,
+        unfiltered_entries_latest.regional_feed_type,
+        unfiltered_entries_latest.uri,
+        unfiltered_entries_latest.base64_url,
+        validation_schedule.base64_url AS schedule_to_use_for_rt_validation_base64_url
```

Comment:
Could we also provide decoded URLs for ease of use? Especially because we use URL-safe encoding which isn't obvious based on our column name (hindsight 2020).

I'm also realizing that `base64url` is somewhat the standard name so maybe we should've called this `base64url_url` 😅

Reply:
Addressed in [e3ada9d](https://github.com/cal-itp/data-infra/pull/2434/commits/e3ada9d537448d246054f6602d50ed65ec2b91e5)